pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
 
     <properties>
-        <eiffel-remrem-generate.version>2.0.12</eiffel-remrem-generate.version>
+        <eiffel-remrem-generate.version>2.0.13</eiffel-remrem-generate.version>

```

Comment:
I just realized now that this is most probably not just a patch fix. It is non-backwards compatible as I understand it, as the way external Event Repositories are handled is negated, right? Previously external Event Repositories where queried by default on lookups, but not they are not. If I've understood this correctly I believe that this new version should be 3.0.0.